terraform/README.md

```diff
@@ -0,0 +1,16 @@
+## Decal VM Terraform configuration
+
+There are two additional files containing sensitive
+information to use with this configuration,
+credentials.auto.tfvars and students.auto.tfvars.
+
+The .tfstate file is also available via more secure channels.
+
+## Setup (untested)
```

Comment:
Why is this untested, didn't you do this to set it up initially?

Reply:
I set terraform up over winter break, and only the one time, so while this all should work I'm disclaiming liability in case it doesn't. If we only use terraform from the decal VM than this is not super relevant.

Reply:
Fair enough, it's not like we'll need to set this up again very soon anyway.